go.mod

```
@@ -6,7 +6,7 @@ toolchain go1.22.9
 
 require (
 	github.com/onsi/ginkgo/v2 v2.22.2
-	github.com/onsi/gomega v1.36.2
+	github.com/onsi/gomega v1.36.3
```

```suggestion
	github.com/onsi/gomega v1.36.2
```

Reverting this so we can only get the updates in the GH Workflow files.